bin/solid-auth-client.js

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env node
+
+const fs = require('fs')
+const path = require('path')
+
+const program = require('commander')
+
+const { version } = require('../package.json')
+
+// helpers
+
+function ns(fn) {
+  return function() {
+    fn.apply(
```

Comment:
There's no need for `apply` anymore in ES6, just use the [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator).